python/ray/air/tests/test_dataset_config.py

```diff
@@ -250,7 +250,7 @@ def checker(shard, results):
         # applying the preprocessor on each epoch.
         assert results[0] == results[1], results
         stats = shard.stats()
-        assert "Stage 1 read->map_batches: 1/1 blocks executed " in stats, stats
+        assert "Stage 1 read->_transform_pandas: 1/1 blocks executed " in stats, stats
```

Comment:
Could we preserve the original high level operation name like this? `map_batches(_transform_pandas)`

Reply:
Is there any way we could use the `Preprocessor` name or representation? If you have many preprocessor I imagine it'd look something like this?

```
Stage 1->_transform_pandas->_transform_numpy->_transform_numpy->_transform_pandas
```

when you'd probably want something like this?

```
Stage 1->OneHotEncoder->Concatenator->TorchVisionPreprocessor->LabelEncoder
```

Reply:
Actually, related to Eric's suggestion, maybe something like

```
MapBatches(my_udf) 
MapBatches(<lambda at 0x12346>)
```

for UDFs and 

```
OneHotEncoder  # Just use preprocessor name
Concatenator
```

for preprocessor?

I imagine it'd be more complex to implement, but I think the UX would be a lot better.

cc @ericl what do you think?